website/source/index.html.erb

```diff
@@ -77,22 +77,22 @@ description: |-
       <div class='g-use-cases'>
         <div>
           <div>
-            <img src='/assets/images/consul-jtbd/connect.png' alt='Connect services'>
-            <h3>Secure Service Communication</h3>
-            <p>Secure and observe communication between your services without modifying their code.</p>
+            <img src='/assets/images/consul-jtbd/kubernetes.png' alt='Upgrade services'>
+            <h3>Consul-Kubernetes Deployments</h3>
```

Comment:
I have some minor reservations on the title here – I wanted to say "Kubernetes Service Mesh" but realized that wouldn't cover service discovery features inside and outside of Kubernetes. No better suggestions though.

Reply:
I'd like to get it out there for Kubcon, but will think about better titles.